src/providers/algolia.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Adobe. All rights reserved.
+ * Copyright 2020 Adobe. All rights reserved.
```

Comment:
```suggestion
 * Copyright 2019 Adobe. All rights reserved.
```

I think we wast to claim the earlier copyright date.